pkg/scannerv4/client/client.go

```diff
@@ -184,31 +183,19 @@ func createGRPCConn(ctx context.Context, o connOptions) (*grpc.ClientConn, error
 func (c *gRPCScanner) GetSBOM(ctx context.Context, ref name.Digest) ([]byte, bool, error) {
 	// verify index report exists for the image
 	hashId := getImageManifestID(ref)
-	_, found, err := c.GetImageIndex(ctx, hashId)
+	ir, found, err := c.GetImageIndex(ctx, hashId)
```

Comment:
confirming we want `GetImageIndex` (ie. require the index already exists) vs `GetOrCreateImageIndex` (ie. create it, if it does not already exist)

Reply:
Yes, the design has the details for this - TL;DR; when we add support for delegated scans, node scans (if we so choose), etc. we DO NOT want this to attempt to index.